src/ServiceManagement/Common/Commands.ScenarioTest/ServiceManagement/ScenarioTests.cs

```diff
@@ -182,6 +182,33 @@ public void TestMigrateAzureVNet()
         [Trait(Category.Service, Category.ServiceManagement)]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         [Trait(Category.AcceptanceType, Category.BVT)]
+        public void TestMigrationNetworkSecurityGroup()
+        {
+            this.RunPowerShellTest("Test-MigrationNetworkSecurityGroup");
+        }
+
+        [Fact]
+        [Trait(Category.Service, Category.ServiceManagement)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
```

Comment:
Note that the other AcceptanceType attribute is redundant.  CheckIn tests are automatically included as BVTs